TaskManagementSystem/main.go

```diff
@@ -7,27 +7,58 @@ import (
 	"time"
 )
 
```

Comment:
I suggest to move constants to global scope as of they are not changing
```suggestion
	const (
		Low = iota + 1
		Medium
		High
	)
```